index.php

```diff
@@ -530,6 +513,7 @@
 
 require_once($template);
```

Comment:
Uh oh, first I thought of a potential candidate for remote-inclusion, `theme_include()` uses `file_exists()` but that may not be enough as escaping untrusted ($_GET, $_POST, $_COOKIE, $_REQUEST, even $_SERVER) is highly recommended or better mandatory.